### PR TITLE
Remove unnecessary point

### PR DESCRIPTION
**Pull request by Mak-Di**

### CODE REVIEW

File: source/localizable/configuring-ember/disabling-prototype-extensions.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ islands.contains('Oahu');
 ### Strings
 
 Strings will no longer have the convenience methods described in the
-[Ember.String API reference.](http://emberjs.com/api/classes/Ember.String.html). Instead,
+[Ember.String API reference](http://emberjs.com/api/classes/Ember.String.html). Instead,
```
Comment (locks) [inline source/localizable/configuring-ember/disabling-prototype-extensions.md]: While you're in here, can you put backticks in `Ember.String`, and a newline after the period please?

Comment (Mak-Di) [inline source/localizable/configuring-ember/disabling-prototype-extensions.md]: @locks  +

**Comment by locks** [comment]
@Mak-Di can you squash the commit, please? Doing it via GitHub interface messes up our process.

**Comment by Mak-Di** [comment]
@locks +

**Comment by locks** [comment]
Thanks for your contribution!

@homu r+

**Pull request merged**